guides/source/debugging_rails_applications.md

```diff
@@ -232,23 +231,31 @@ irb(main):003:0> Article.pamplemousse
 => #<Comment id: 2, author: "1", body: "Well, actually...", article_id: 1, created_at: "2018-10-19 00:56:10", updated_at: "2018-10-19 00:56:10">
 ```
 
-Below each database statement you can see arrows pointing to the specific source filename (and line number) of the method that resulted in a database call. This can help you identify and address performance problems caused by N+1 queries: single database queries that generates multiple additional queries.
+Below each database statement you can see arrows pointing to the specific source filename (and line number) of the method that resulted in a database call e.g. `↳ app/models/article.rb:5`.
+
+This can help you identify and address performance problems caused by N+1 queries: i.e. single database queries that generate multiple additional queries.
 
-Verbose query logs are enabled by default in the development environment logs after Rails 5.2.
+Verbose query logs are [enabled by default](/configuring.html#config-active-record-verbose-query-logs) in the development environment logs.
```

Comment:
```suggestion
Verbose query logs are [enabled by default](configuring.html#config-active-record-verbose-query-logs) in the development environment logs.
```

I think the slash could potentially lose any version tag that proceeds the file path (all guides are at the same directory-level).

Reply:
Change applied and forced-pushed: there should not be a single squashed commit.